x-pack/plugins/fleet/public/components/platform_selector.tsx

```diff
@@ -37,19 +43,37 @@ const CommandCode = styled.pre({
   overflow: 'auto',
 });
 
-const K8S_COMMAND = `kubectl apply -f elastic-agent-managed-kubernetes.yaml`;
-
 export const PlatformSelector: React.FunctionComponent<Props> = ({
   linuxCommand,
   macCommand,
   windowsCommand,
   linuxDebCommand,
   linuxRpmCommand,
+  k8sCommand,
   isK8s,
+  isManaged,
+  enrollToken,
+  isFleet,
```

Comment:
I think that this naming is not very clear, could you use something more descriptive like `isFleetServer`?